pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -331,3 +335,18 @@ type UProbeSpec struct {
 	// Selectors to apply before producing trace output. Selectors are ORed.
 	Selectors []KProbeSelector `json:"selectors"`
 }
+
+type ListSpec struct {
+	// Name of the list
+	Name string `json:"name"`
+	// +kubebuilder:validation:Optional
+	// Values of the list
+	Values []string `json:"values"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=syscalls;generated_syscalls;generated_ftrace
+	// Indicates the type of the list values.
+	Type string `json:"type"`
```

Comment:
As far as I can tell from the code, `generate_syscalls` will generate all syscalls (and ignore the `generate` argument). Is there a plan to add pattern matching in the `generate` argument as well?

Reply:
I saw the use case for that in your recent document, so yes that can be added,
also I'm thinking of renaming the 'generate' field to 'pattern'

Reply:
I personally like `pattern` but naming is subjective so I'm fine with either option :)